pkg/cacheutil/redis_client_test.go

```diff
@@ -136,3 +136,84 @@ func TestRedisClient(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateRedisConfig(t *testing.T) {
+	assert_ok := func(t *testing.T, _ interface{}, e error) { testutil.Ok(t, e) }
```

Comment:
This is interesting. It's not similar to other table-tests - we typically have an `error` field or `bool` to signal whether any error is expected. I suggest adding `err bool` to `tests` struct instead && just call `testutil.{Ok,NotOk}` there instead of these functions.

Reply:
I was originally doing some slightly fancier assertion and validation of the config object but it eventually ended up unnecessary. I've removed it for the bool now.